main.go

```diff
@@ -150,20 +150,32 @@ func handlePullRequest(w http.ResponseWriter, body []byte, git Git, githubClient
 		message := fmt.Sprintf("Getting commits for PR %s/%s#%d failed", pullRequestEvent.Repository.Owner, pullRequestEvent.Repository.Name, pullRequestEvent.IssueNumber)
 		return ErrorResponse{err, http.StatusBadGateway, message}
 	}
+	if includesFixupCommits(commits) {
+		pr, _, err := githubClient.PullRequests.Get(pullRequestEvent.Repository.Owner, pullRequestEvent.Repository.Name, pullRequestEvent.IssueNumber)
+		if err != nil {
+			message := fmt.Sprintf("Getting PR %s/%s#%d failed", pullRequestEvent.Repository.Owner, pullRequestEvent.Repository.Name, pullRequestEvent.IssueNumber)
+			return ErrorResponse{err, http.StatusBadGateway, message}
+		}
+		_, _, err = githubClient.Repositories.CreateStatus(pullRequestEvent.Repository.Owner, pullRequestEvent.Repository.Name, *pr.Head.SHA, &github.RepoStatus{
+			State:       github.String("pending"),
+			Description: github.String("This PR contains commits that need to be squashed with !squash before merging"),
```

Comment:
can we shorten this text a bit? 
<img width="775" alt="screen shot 2015-10-13 at 19 31 53" src="https://cloud.githubusercontent.com/assets/3438/10461245/2230af6e-71e1-11e5-8489-7019143291ab.png">